source/postprocess/boundary_velocity_residual_statistics.cc

```diff
@@ -42,6 +42,13 @@ namespace aspect
           data_lookup = std_cxx14::make_unique<Utilities::StructuredDataLookup<dim>>(dim, scale_factor);
           data_lookup->load_file(data_directory + data_file_name, this->get_mpi_communicator());
         }
+
+      else if (use_surface_strain_rate_data)
+        {
+          strain_rate_data_lookup = std_cxx14::make_unique<Utilities::StructuredDataLookup<dim> >((dim-1)*(dim-1), scale_factor);
+          strain_rate_data_lookup->load_file(data_directory + data_file_name, this->get_mpi_communicator());
+        }
```

Comment:
So at the moment you can only use this postprocessor for the velocity or the strain rate, but not both? This suggests to me we should instead have a new `boundary_strain_rate_residual_statistics` so that you could output both at the some time. Or is there a reason you would never want to do both at the same time?